RELEASE.md

```diff
@@ -0,0 +1,113 @@
+# web3.js Release Guidelines
+
+To provide the community with latest features and patches as soon as possible, we follow the `ship-it` philosophy along with [conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/).
```

Comment:
As a practical matter, this isn't being followed. Maybe this could be a separate proposal?

Reply:
gotcha thanks, will remove from this doc for now